lib/zoom/actions/report.rb

```diff
@@ -40,9 +40,9 @@ def meeting_polls_report(*args)
       end
 
       def webinar_details_report(*args)
-        # TODO: implement webinar_details_report
-        # params = Zoom::Params.new(Utils.extract_options!(args))
-        raise Zoom::NotImplemented, 'webinar_details_report is not yet implemented'
+        params = Zoom::Params.new(Utils.extract_options!(args))
+        params.require(:id).permit(%i[page_size next_page_token])
+        Utils.parse_response self.class.get("/report/webinars/#{params[:id]}", query: params.except(:id), headers: request_headers)
```

Comment:
Update to match id name change
```suggestion
        Utils.parse_response self.class.get("/report/webinars/#{params[:webinar_id]}", headers: request_headers)
```